config/samples/09-federation/posts-subgraph.yaml

```diff
@@ -0,0 +1,133 @@
+# Create Service & Deployment of Posts API
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: posts
+  labels:
+    app: posts
+spec:
+  ports:
+    - name: http
+      port: 4002
+      targetPort: 4002
+  selector:
+    app: posts
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: posts
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: posts
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: posts
+        version: v1
+    spec:
+      containers:
+        - image: zalbiraw/go-api-test-service
+          imagePullPolicy: Always
+          name: users
+          command: ["./posts/server"]
+          env:
+            - name: PORT
+              value: "4002"
+---
+apiVersion: tyk.tyk.io/v1alpha1
+kind: SubGraph
+metadata:
+  name: posts-subgraph
+spec:
+  subgraph:
```

Comment:
Do we need this `subgraph` field? Can we just omit it and specify `schema` and `sdl` as `spec.schema` and `spec.sdl` instead of `spec.graphql.schema` and `spec.graphql.sdl`? @komalsukhani @andrei-tyk ?

Reply:
updated 👍